src/server/enrollment-system.js

```diff
@@ -241,19 +241,19 @@ function veteranToSpouseInfo(veteran) {
  */
 function resourceToIncomeCollection(resource) {
   const incomeCollection = [];
-  if (resource.grossIncome) {
+  if (resource.grossIncome !== undefined) {
```

Comment:
I may be misunderstanding this, but `!isNaN(null) == true`, so null would still get through with that.